pkg/operator/controllers/internetchecker/internetchecker_controller.go

```diff
@@ -24,21 +25,29 @@ import (
 	"github.com/Azure/ARO-RP/pkg/operator/controllers"
 )
 
+const (
+	// schedule the check every 5m
```

Comment:
as implemented, doesn't the check take up to 2.5 minutes, then it'll wait for 5?  The comment seems misleading and I'm not sure that's the implemented functionality makes sense anyway?  I'd be inclined to reschedule after one minute like everything else does.